data/input_files/run_serac_tests.sh

```diff
@@ -0,0 +1,118 @@
+#! /bin/bash
+#
+# desc: runs all inputs on serac moves their summary.json files
+# to the tests integration directory
+#
+# notes:
+#  - this is meant to run in ./serac_repo/data/input_files
+#  - SERAC_EXE can be found in a build directory under "bin"
+#  - you can run without sbatch with IS_PARALLEL=0
+#  - KEEP_OUTPUTFILE for debugging
+#
+# how to run:
+#  - serially: ./run_serac_tests.sh
+#  - in parallel: sbatch ./run_serac_tests.sh
+#
+
+#SBATCH -J run_serac_tests
+#SBATCH -t 2:00:00
+#SBATCH -p pbatch
+#SBATCH -N 1
+#SBATCH -n 2
+
+if [ $# -ne 3 ] ; then
+    echo "(sbatch) ./run_serac_tests.sh <SERAC_EXE> <IS_PARALLEL> <KEEP_OUTPUTFILE>"
+    exit 1
+fi
+
+SERAC_EXE=$1
+IS_PARALLEL=$2
+KEEP_OUTPUTFILE=$3
+
+# set dir vars
+INPUT_DIR=$(pwd)
+BASE_SERAC_OUTPUT_DIR=$(pwd)/../output_files
+SERAC_REPO_DIR="${INPUT_DIR}/../../"
+BASE_INTEGRATION_DIR="${SERAC_REPO_DIR}/tests/integration"
+cd ${INPUT_DIR}
+
+# if output directory exists, delete it
+if [ ! -f ${BASE_SERAC_OUTPUT_DIR} ] ; then
+    rm -rf ${BASE_SERAC_OUTPUT_DIR}
+fi
+
+# create output directory
+mkdir -p ${BASE_SERAC_OUTPUT_DIR}
+
+shopt -s globstar
+for f in **/*.lua ; do
+
+    # skip default
+    if [ ${f} = "default.lua" ] ; then
```

Comment:
Instead of skipping default, you should just skip if the directory already exists and add an option to override existing.